### ISSUE TITLE
Support for other issue types

### ISSUE THREAD

**Issue by holowinski** [ulfschneider/breakdown#107]
Hi.

It would be great to have support for issue types other than Epic,Story,Sub-task.

In some of the projects I'm handling, we have a different set of issue types, i.e:

```
Epic
  Story
    Sub-task
    Documentation sub task
  Task
    Tech-sub-task
  Bug
```

This is currently not supported in breakdown, and I understand that this might be difficult to implement, but it would be a significant extension to the plugin's capabilities, if it would support arbitrary issue types trees it would allow handling more kinds of projects.

**Comment by ulfschneider** [comment]
I´m aiming to support at least the Task issue type. To have customizable issuetypes is probably not possible because to my understanding, the grammar behind needs to be configured upfront and doesn´t allow such variables.

**Comment by holowinski** [comment]
Yeah, I had a quick look at the implementation and had the same feeling.

Would you consider adding Bugs as well ?
In my case we have bugs which might or might not be attached to particular Epics / Stories which currently are not visible at all in breakdown.
